fitnessllm_dataplatform/utils/logging_utils.py

```
@@ -41,0 +1,131 @@
+"""Logging utilities."""
+import json
+import logging
+from datetime import datetime
+from logging import Logger
+from typing import Any, Dict, Optional
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)